assets/js/state/sagas/user.test.js

```diff
@@ -4,16 +4,42 @@ import {
   authClient,
   getAccessTokenFromStore,
   getRefreshTokenFromStore,
+  storeRefreshToken,
+  storeAccessToken,
 } from '@lib/auth';
 import {
   setAuthInProgress,
   setAuthError,
   setUser,
   setUserAsLogged,
 } from '@state/user';
-import { performLogin } from './user';
+import { networkClient } from '@lib/network';
+import { performLogin, clearUserAndLogout } from './user';
 
 const axiosMock = new MockAdapter(authClient);
+const networkClientAxiosMock = new MockAdapter(networkClient);
+
+describe('user actions saga', () => {
+  beforeEach(() => {
+    axiosMock.reset();
+    jest.spyOn(console, 'error').mockImplementation(() => null);
+  });
+
+  afterEach(() => {
+    /* eslint-disable-next-line */
+    console.error.mockRestore();
+  });
+
+  it('should clear the storage in clearUserAndLogout saga', async () => {
```

Comment:
I know it's a bit nitpicky, but it always helps to see a "when" or an "after" in a test description